docs/index.asciidoc

```diff
@@ -138,6 +138,7 @@ See <<plugins-{type}s-{plugin}-object_order>>.
 returns multiple columns, the data is stored as a JSON object within the field.
 <5> Takes data from the JSON object and stores it in top-level event fields for
 easier analysis in Kibana.
+<6> Local lookup queries could be use also prepared statements.
```

Comment:
@karenzone Is this an issue with out of order callouts? The `<6>` is defined before the `<5>` and I seem to recall that callouts are auto-numbered(?), and ignored the number given in the table other than as a label to refer back to

Reply:
Yes, @robbavey. You are correct!
@andsel The callouts must be issued in numerical order.  Also, since our conversion to asciidoctor, we can't reuse a number (see `<5>`, for example). In other words, what was there worked before we converted to asciidoctor. It hasn't been quite right since we converted.  Please let me know if you have questions.

Reply:
If you want entries for both `<5>`, you can define once and assume the user will get it.  Or, you can define as <5> and <6>, and reorder the other items accordingly.

As long as you define your callouts in order in your code sample, and don't repeat numbers, you should be fine.

Reply:
@karenzone I've fixed the number ordering, but I don't know how to fix the issue related to double use of same callout

Reply:
@andsel @karenzone How about having a single callout in the comment above the `add_field` settings - I'm not quite sure why the docs originally only had callouts on two of the three `add_field`, and putting it in the comment might work as a way of grouping the settings

Reply:
I agree with you, and I would add the callout to the first of the `add_field` definitions